client/src/components/advancedSelectWidget/AdvancedSelect.tsx

```diff
@@ -139,7 +140,8 @@ const AdvancedSelect = ({
   fields,
   handleAddItem,
   handleRemoveItem,
-  createEntityComponent
+  createEntityComponent,
+  autoComplete = "on"
```

Comment:
I would leave this uninitialized by default (`undefined` props are left out by React), so:
```javascript
  autoComplete
```